### DISCUSSION TITLE
API to specify wiki permissions

### DISCUSSION THREAD

**Discussion by Gby56** [community/community#51582] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hi!
I'd like to report there is no API to set the special checkbox "Restrict editing to collaborators only" for wikis in repositories.
I think it's `wiki_access_to_pushers`

This would be good to set the setting on a wide-scale governance of a company's repositories.
For reference, it is also a pain that some projects have https://github.com/orgs/community/discussions/34991

https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis

https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#update-a-repository

**Comment by Aldekein** [comment]
At least GET API for this parameter is required to protect the security perimeter of GitHub organization: there's no way currently to check if someone did ever deselect _Restrict editing to collaborators only_ option other than visiting settings page for each repository manually.
